src/test/java/org/broadinstitute/hellbender/tools/walkers/sv/SVStratifyIntegrationTest.java

```diff
@@ -0,0 +1,208 @@
+            }
+        }
+        Assert.assertEquals(numVcfs, 7);
+        final int numInputRecords = VariantContextTestUtils.readEntireVCFIntoMemory(inputVcfPath).getRight().size();
+        Assert.assertEquals(totalRecords, numInputRecords);
+    }
+
+    @Test
+    public void testBwaMeltCohortSingleOutput() {
+        final File outputDir = createTempDir("stratify");
+        final File outputFile = outputDir.toPath().resolve("out.vcf.gz").toFile();
+        final String inputVcfPath = getToolTestDataDir() + "bwa_melt.chr22.vcf.gz";
+        final String configFile = getToolTestDataDir() + "test_config.tsv";
+
+        final String segdupFile = getToolTestDataDir() + "hg38.SegDup.chr22.bed";
+        final String segdupName = "SD";
+        final String repeatmaskerFile = getToolTestDataDir() + "hg38.RM.chr22_subsampled.bed";
+        final String repeatmaskerName = "RM";
+
+        final ArgumentsBuilder args = new ArgumentsBuilder()
+                .addOutput(outputFile)
+                .add(SVStratificationEngineArgumentsCollection.STRATIFY_CONFIG_FILE_LONG_NAME, configFile)
+                .add(SVStratificationEngineArgumentsCollection.CONTEXT_NAME_FILE_LONG_NAME, segdupName)
+                .add(SVStratificationEngineArgumentsCollection.CONTEXT_INTERVAL_FILE_LONG_NAME, segdupFile)
+                .add(SVStratificationEngineArgumentsCollection.CONTEXT_NAME_FILE_LONG_NAME, repeatmaskerName)
+                .add(SVStratificationEngineArgumentsCollection.CONTEXT_INTERVAL_FILE_LONG_NAME, repeatmaskerFile)
+                .add(SVStratificationEngineArgumentsCollection.OVERLAP_FRACTION_LONG_NAME, 0.5)
+                .add(StandardArgumentDefinitions.SEQUENCE_DICTIONARY_NAME, GATKBaseTest.FULL_HG38_DICT)
+                .add(StandardArgumentDefinitions.VARIANT_LONG_NAME, inputVcfPath);
+
+        runCommandLine(args, SVStratify.class.getSimpleName());
+
+        final List<File> outputFiles = Lists.newArrayList(outputDir.listFiles()).stream().filter(VcfUtils::isVariantFile).collect(Collectors.toUnmodifiableList());
+        Assert.assertEquals(outputFiles.size(), 1);
+        Assert.assertEquals(outputFiles.get(0).getAbsolutePath(), outputFile.getAbsolutePath());
+        final Pair<VCFHeader, List<VariantContext>> inputVcf = VariantContextTestUtils.readEntireVCFIntoMemory(inputVcfPath);
+        final Pair<VCFHeader, List<VariantContext>> outputVcf = VariantContextTestUtils.readEntireVCFIntoMemory(outputFile.getAbsolutePath());
+        Assert.assertEquals(outputVcf.getRight().size(), inputVcf.getRight().size());
+    }
+
+    @Test(expectedExceptions = GATKException.class)
+    public void testBwaMeltCohortRedundant() {
+        final File outputDir = createTempDir("stratify");
+        final String inputVcfPath = getToolTestDataDir() + "bwa_melt.chr22.vcf.gz";
+        final String configFile = getToolTestDataDir() + "test_config_redundant.tsv";
```

Comment:
What happens if the config conflicts with the cmdline args?

Reply:
It'll get caught in the `Stratrum` class and throw an error